.github/workflows/update-dispatch.yml

```diff
@@ -34,11 +34,15 @@ jobs:
             skip: true
         update:
           - source: "versions/0_1"
-            cmd: "nix run --override-input versions ./versions/0_1 .#scripts-repo-flake-update"
+            cmd: "nix run .#scripts-repo-flake-update 0_1"
+          - source: "versions/0_2"
```

Comment:
`0_2` doesn't exist on the 0.1 line. Is this a mistake?

Reply:
```suggestion
          - source: "versions/0_1"
```

Reply:
it's just because i copied all of `.github` from develop. this won't do anything here unless we trigger it manually